### ISSUE TITLE
Firejail profile for transmission-cli excludes a dependency

### ISSUE THREAD

**Issue by R1kaB3rN** [netblue30/firejail#5491]
### Description
Unable to firejail transmission-cli and download/upload content, presumably due to the firejail profile excluding a dependency related to openssl. Running without firejail, I can confirm that I am able to successfully download/upload content.

### Steps to Reproduce

1. open a terminal emulator
2. $ zsh
3. $ firejail transmission-cli -ep _someMagnetLink_

### Log
Terminal output:
```
transmission-cli 3.00 (bb6b5a062e)

[2022-11-29 18:00:34.434] Transmission 3.00 (bb6b5a062e) started
[2022-11-29 18:00:34.435] RPC Server: Adding address to whitelist: 127.0.0.1
[2022-11-29 18:00:34.435] RPC Server: Adding address to whitelist: ::1
[2022-11-29 18:00:34.435] UDP: Failed to set receive buffer: requested 4194304, got 425984
[2022-11-29 18:00:34.435] UDP: Please add the line "net.core.rmem_max = 4194304" to /etc/sysctl.conf
[2022-11-29 18:00:34.435] UDP: Failed to set send buffer: requested 1048576, got 425984
[2022-11-29 18:00:34.435] UDP: Please add the line "net.core.wmem_max = 1048576" to /etc/sysctl.conf
[2022-11-29 18:00:34.435] UDP: Failed to set receive buffer: requested 4194304, got 425984
[2022-11-29 18:00:34.435] UDP: Please add the line "net.core.rmem_max = 4194304" to /etc/sysctl.conf
[2022-11-29 18:00:34.435] UDP: Failed to set send buffer: requested 1048576, got 425984
[2022-11-29 18:00:34.435] UDP: Please add the line "net.core.wmem_max = 1048576" to /etc/sysctl.conf
[2022-11-29 18:00:34.435] DHT: Reusing old id
[2022-11-29 18:00:34.435] DHT: Bootstrapping from 120 IPv4 nodes
[2022-11-29 18:00:34.435] DHT: Bootstrapping from 21 IPv6 nodes
[2022-11-29 18:00:34.484] Saved "/home/foo/.config/transmission/torrents/12345.torrent"
[2022-11-29 18:00:34.766] Port Forwarding (NAT-PMP): initnatpmp succeeded (0)
[2022-11-29 18:00:34.767] Port Forwarding (NAT-PMP): sendpublicaddressrequest succeeded (2)
Progress: 0.0%, dl from 0 of 0 peers (0 kB/s), ul t[2022-11-29 18:00:38.796] Port Forwarding: State changed from "Not forwarded" to "Starting"
[2022-11-29 18:00:38.798] web: will verify tracker certs using envvar CURL_CA_BUNDLE: none
[2022-11-29 18:00:38.798] web: NB: this only works if you built against libcurl with openssl or gnutls, NOT nss
[2022-11-29 18:00:38.798] web: NB: invalid certs will show up as 'Could not connect to tracker' like many other errors
[2022-11-29 18:00:39.150] bar.mkv: Tracker gave HTTP response code 0 (No Response)
[2022-11-29 18:00:39.150] bar.mkv: Retrying announce in 20 seconds.
[2022-11-29 18:00:39.491] tr_crypto_utils: OpenSSL error: error:12800067:DSO support routines::could not load the shared library
```

### Environment
Distribution: Arch Linux
Kernel: 6.0.10-arch2-1
Firejail version: 0.9.70
Firejail profile: /etc/firejail/transmission-cli.profile 
Transmission-CLI version: 3.00 (bb6b5a062e)

**Comment by rusty-snake** [comment]
Fixed by 72ece92ea303bf272438ef62815a9c9ce3d71a62

**Comment by kmk3** [comment]
Duplicate of #5211

**Comment by kmk3** [comment]
Closing since it appears to have already been fixed in #5213.

Feel free to reopen if you can reproduce it with the current development
version.
